src/test/browser-config.js

```diff
@@ -14,8 +14,7 @@ function getPatterns (ctx) {
   }
 
   return [
-    'test/browser.js',
-    'test/**/*.spec.js'
+    'test/browser.js'
```

Comment:
This change as it is will break the assumption on tests everywhere.

To make this happen, we need to make it consistent (either use .spec for node and browser or for none) and we need updated docs.